txt2html/PKGBUILD

```diff
@@ -4,6 +4,7 @@
 pkgname=txt2html
 pkgver=2.5201
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `txt2html`. Therefore this change should be unnecessary.